docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
     - MYSQL_USER=mysqldb
     - MYSQL_PASSWORD=mysqldb
     ports:
-    - "127.0.0.1:3306:3306"
+    - "127.0.0.1:3307:3306"
```

Comment:
Is this a fix to get it to work in CI?

Reply:
Yes. On the first CI run, the step that brings up the docker containers for the Integration Test run failed because `127.0.0.1:3306` was already taken. We'll see if just bumping by one allows us to bind correctly.

Reply:
Actually, I already know that the integration tests have gotten past this point in the setup, so that's progress!

Reply:
Next, we'll need to verify that the MySql test passes. Locally I was able to run the test and it worked correctly so I don't expect hiccups, but we'll see.

Reply:
thanks!